src/frontend/Komponenter/Oppgavebenk/OppgaveFiltrering.tsx

```diff
@@ -62,6 +62,24 @@ interface Feil {
 
 const initFeilObjekt = {} as Feil;
 
+const oppgaveRequestMedDefaultEnhet = (
+    oppgaveRequest: IOppgaveRequest,
+    harSaksbehandlerStrengtFortroligRolle: boolean
+): IOppgaveRequest => {
+    if (harSaksbehandlerStrengtFortroligRolle) {
+        return {
+            ...oppgaveRequest,
+            enhet: FortroligEnhet.VIKAFOSSEN,
+        };
+    } else {
+        const enhet = oppgaveRequest.enhet;
+        return {
+            ...oppgaveRequest,
+            enhet: enhet && enhet !== FortroligEnhet.VIKAFOSSEN ? enhet : undefined,
```

Comment:
Det skal vel ikke skje at enhet er Vikafossen i localstorage? Med mindre brukeren mister vikafossen-tilgangen sin.
Vi lagrer jo localstorage på innlogget bruker.

Reply:
Nei det skal vel ikke det. Det er spesielle brukere for kode6-tilgangene (de logger inn med en annen bruker)
Skal jeg fjerne sånn att det kun er `return oppgaveRequest` ?